spacy/cli/debug_config.py

```diff
@@ -1,7 +1,7 @@
 from typing import Optional, Dict, Any, Union, List
 from pathlib import Path
 from wasabi import msg, table
-from thinc.api import Config
+from thinc.config import Config
```

Comment:
Why did this need to change? In general we prefer importing from `thinc.api`.

Reply:
We don't, that was a mistake on my part. Reverted in next commit.